view/templates/hovercard.tpl

```diff
@@ -22,8 +22,12 @@
 				{{* @todo we have two different photo menus one for contacts and one for items at the network stream. We currently use the contact photo menu, so the items options are missing We need to move them *}}
 				<div class="hover-card-actions-social">
 					{{if $profile.actions.pm}}<a class="btn btn-labeled btn-primary btn-sm add-to-modal" href="{{$profile.actions.pm.1}}" aria-label="{{$profile.actions.pm.0}}"><i class="fa fa-envelope" aria-hidden="true" title="{{$profile.actions.pm.0}}"></i><span class="sr-only">{{$profile.actions.pm.0}}</span></a>{{/if}}
+
+					{{if $profile.addr && !$profile.actions.self.0}}<a class="btn btn-labeled btn-primary btn-sm" href="{{$profile.actions.mention.2}}" aria-label="{{if $profile.actions.group.2==1}}{{$profile.actions.mention.1}}{{else}}{{$profile.actions.mention.0}}{{/if}}" title="{{if $profile.actions.group.2==1}}{{$profile.actions.mention.1}}{{else}}{{$profile.actions.mention.0}}{{/if}}"><i class="fa fa-pencil-square-o" aria-hidden="true"></i></a>{{/if}}
```

Comment:
> After you moved the `self` key to the Hovercard template variable.

you mean `'self'     => $contact['self'] ?? false,` instead of `'self'     => [$contact['self'] ?? false, true],`?

I tried this before i pushed the commit. It did not work.
And i tried it again, the "mention"-button is shown on my own profiles hovercard.
With my solution, it is not shown.

Or am i doing it wrong?

Reply:
I don't understand this... 
The only solution, which shows the mention-button on others and hide it on my own profile is the following:

in Contact.php 
`'self'     => [$self, true],` in both `$menu = [` parts (where $self is 0 or 1 or its true or false) 
and in hovercard.tpl
`!$profile.actions.self.0` in the if-clause... 

i played around with all possible combinations... nothing else works (event not, what you suggested).
Would be glad, if you can explain this. I really want to understand what happens here.

Reply:
I'll pull your branch, see what I can do with it.

Reply:
I was able to enable my changes and I couldn't resist simplifying the template further by moving group-based conditions to the PHP logic in `Model\Contact::photoMenu`. If you want to keep working on your branch, be sure to pull it first.

Reply:
> I was able to enable my changes and I couldn't resist simplifying the template further by moving group-based conditions to the PHP logic in `Model\Contact::photoMenu`. If you want to keep working on your branch, be sure to pull it first.

yes, i will do.
but which branch from where i should pull?

Reply:
i see!

Did you remove the "View group"-button?
Or is it just removed, because i should reimplement it using the new logic?

Reply:
I see the new logic, and improved it a bit, to show the group-icon instead of cloud-icon for "Show group" in hovercard for groups.

Reply:
Did you pull your branch first before committing your new changes? They don't show in the PR yet.

Reply:
Ok... i did...

Will check it at home again.